torchinductor/scheduler.py

```diff
@@ -382,6 +383,78 @@ def get_priority(self):
             return len(self.group) - 1
 
 
+class FusedSchedulerNode(BaseSchedulerNode):
+    def __init__(self, scheduler: "Scheduler", snodes):
```

Comment:
Call `super().__init__()` or add a comment explaining why you're not calling it.

Reply:
I'm not calling it since I don't think any of the things done in the baseschedulernode class are needed? Is that a valid reason :)

Reply:
Yeah, just document that to it is clear that this is intentional.